### PR TITLE
ops: docker compose up config | wip: still needs embeddings params 

### PR DESCRIPTION
**Pull request by skeptrunedev**

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -111,76 +110,74 @@ services:
...
-      - OIDC_AUTH_REDIRECT_URL=${OIDC_AUTH_REDIRECT_URL}
-      - OIDC_ISSUER_URL=${OIDC_ISSUER_URL}
-      - GPU_SERVER_ORIGIN=${GPU_SERVER_ORIGIN}
-      - AWS_REGION=${AWS_REGION}
-      - SPARSE_SERVER_ORIGIN=${SPARSE_SERVER_ORIGIN}
-      - EMBEDDING_SERVER_ORIGIN=${EMBEDDING_SERVER_ORIGIN}
-      - RERANKER_SERVER_ORIGIN=${RERANKER_SERVER_ORIGIN}
-      - UNLIMITED=${UNLIMITED}
+      - HF_TOKEN=${HF_TOKEN}
+      - API_KEY=${OPENAI_API_KEY}
```
Comment (skeptrunedev) [inline docker-compose.yml:130]: @fastfists why would we not replace this with TEI?

**Review by skeptrunedev — COMMENTED** [review]
WIP: still need to figure out what we are doing for the embedding vectors here

File: docker-compose.yml (hunk 1 above)
Comment (fastfists) [reply docker-compose.yml:130]: We could but this is 3x simpler than using TEI, TEI requires 4 services, this is just 1.

**Comment by skeptrunedev** [comment]
Will re-open at correct time once there is another thing to review `

**Pull request closed without merging**